cli/tests/089_run_allow_list.ts

```diff
@@ -7,7 +7,9 @@ try {
 }
 
 const proc = Deno.run({
-  cmd: ["cat", "089_run_allow_list.ts"],
+  cmd: Deno.build.os === "windows"
+    ? ["cmd", "/c", "type", "089_run_allow_list.ts"]
+    : ["cat", "089_run_allow_list.ts"],
```

Comment:
I don't like doing these conditionals. Let me know if you can think of something that exists on all three operating systems by default.

Reply:
`curl` exists on all three now AFAIK. Or maybe `echo`?

Reply:
Oh, `echo` is good. Thanks!